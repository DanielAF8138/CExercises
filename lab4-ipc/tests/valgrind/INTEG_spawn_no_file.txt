==3308219== Memcheck, a memory error detector
==3308219== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==3308219== Using Valgrind-3.22.0 and LibVEX; rerun with -h for copyright info
==3308219== Command: ../ipc -s shasum ../data/f5.txt
==3308219== 
shasum: ../data/f5.txt: No such file or directory
shasum(../data/f5.txt) = ''
==3308219== 
==3308219== HEAP SUMMARY:
==3308219==     in use at exit: 0 bytes in 0 blocks
==3308219==   total heap usage: 4 allocs, 4 frees, 8,549 bytes allocated
==3308219== 
==3308219== All heap blocks were freed -- no leaks are possible
==3308219== 
==3308219== For lists of detected and suppressed errors, rerun with: -s
==3308219== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
