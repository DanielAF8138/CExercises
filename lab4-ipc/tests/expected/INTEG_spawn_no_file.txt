shasum(../data/f5.txt) = ''
